packages/installer/installer.sh

```diff
@@ -265,8 +265,12 @@ install_grub()
     mkdir ${TARGET}/sys || true
     mkdir ${TARGET}/tmp || true
     mount -t proc proc ${TARGET}/proc
+
     mount --rbind /dev ${TARGET}/dev
+    mount --make-rslave ${TARGET}/dev
+
     mount --rbind /sys ${TARGET}/sys
```

Comment:
I usually do a `mount -t sysfs /sys ${TARGET}/sys` probably this avoind then having to `umount -R ${TARGET}/sys`.

Reply:
yup that works too indeed, but generally I'd like to get rid of this block entirely, it is mostly there because of #31